common/src/main/scala/io/phdata/clients/LDAPClient.scala

```diff
@@ -38,3 +38,29 @@ trait LDAPClient[F[_]] {
 
   def search(filter: String): F[MemberSearchResult]
 }
+
+trait ProvisioningLDAPClient[F[_]] extends LDAPClient[F] {
+
+  def validateUser(username: String, password: Password): OptionT[F, LDAPUser]
```

Comment:
Are you sure about this one? It surprises me a little bit. Maybe this can be done with both, I'd think a user could bind to the global catalog but not the other. Maybe it's just groups that aren't synced to the standard connection.

Reply:
well to be honest I'm not sure, I just thought that it should be part of `ProvisioningLDAPClient`